### DISCUSSION TITLE
可否添加记忆每个输入方案简繁切换的功能？

### DISCUSSION THREAD

**Discussion by alrcatraz** [rime/home#1741] [Ideas]
因为Rime支持拼音输入，因此我正试图将它作为默认的汉语输入法。我目前使用简体字拼音和粤语拼音，未来还可能需要使用繁体中文拼音，请问可否添加【记忆输入方案+简繁转换设置】组合的功能？这样我就可以在切换输入方案时快速呼出我需要的语言和输入方式，谢谢！

**Comment by lotem** [comment]
輸入方案用到的開關，在輸入方案裏定義。

哪些開關的狀態（通過方案選單切換後）自動保存，由這段代碼指定：

https://github.com/rime/rime-prelude/blob/3803f09458072e03b9ed396692ce7e1d35c88c95/default.yaml#L22

不太明確你的需求。聽上去好像跟自動保存狀態無關。

如果要在切到某個輸入方案時設置開關的初始狀態，用 `reset:` 0 或 1 。

**Reply by alrcatraz** [reply]
多谢指点！
有没有图形化的窗口可以设置这些部分呢？

**Comment by kionz** [comment]
代码编辑器

Alrcatraz ***@***.***> 于 2024年12月7日周六 09:18写道：

**Comment by alrcatraz** [comment]
手动改yaml文件确实可行，但是一旦发生错误，整个输入法都会无法使用，如果有GUI界面就可以避免这个风险。

对于我自己实现我的需求，暂时放弃了全拼，先用粤语拼音、luna-pinyin和luna-pinyin-simp三个组合来实现了。
还是非常感谢各位大佬的热心帮助！

**Reply by lotem** [reply]
不用自己改……有按這個方法寫好的，luna-pinyin 和 luna-pinyin-simp 就是。

**Reply by yrljroli** [reply]
十分建议在修改任何配置文件时、先行备份。

不仅限于<u>rime</u>相关。
